test/parallel/test-net-connect-options-fd.js

```diff
@@ -33,8 +33,9 @@ const forAllClients = (cb) => common.mustCall(cb, CLIENT_VARIANTS);
 
 // Test Pipe fd is wrapped correctly
 {
-  const prefix = path.relative(`${common.PIPE}-net-connect-options-fd`,
-                               `${__dirname}-net-connect-options-fd`);
+  // Using relative path on osx if the path is greaterthan 108 chars
+  // an AssertionError: -48 is thrown
```

Comment:
@Trott 108 should be the right number on Linux and almost certainly other systems too, quote `unix(7)`:
```c
struct sockaddr_un {
    sa_family_t sun_family;               /* AF_UNIX */
    char        sun_path[108];            /* pathname */
};
```

It’s [this](https://github.com/nodejs/node/blob/427125491f3516fc80030f16e1536c3fa30ac929/deps/uv/src/unix/pipe.c#L70-L71) bit of code in libuv that truncates the path to that size. The actual value for EADDRINUSE **is** OS-dependent; for example, for me on Linux it’s 98 (which libuv forwards as -98).

See also #12708, I think that’s the same kind of problem.